src/sas/qtgui/Utilities/Reports/ReportDialog.py

```diff
@@ -4,37 +4,37 @@
 import logging
 import traceback
 from xhtml2pdf import pisa
+from typing import Optional
 
 from PyQt5 import QtWidgets, QtCore
 from PyQt5 import QtPrintSupport
 
 import sas.qtgui.Utilities.GuiUtils as GuiUtils
 import sas.qtgui.Utilities.ObjectLibrary as ObjectLibrary
 
-from sas.qtgui.Utilities.UI.ReportDialogUI import Ui_ReportDialogUI
+from sas.qtgui.Utilities.Reports.UI.ReportDialogUI import Ui_ReportDialogUI
+from qtgui.Utilities.Reports.reportdata import ReportData
```

Comment:
I get an error from this line when trying to run SasView: `ModuleNotFoundError: No module named 'qtgui'`. Should this be `sas.qtgui.Utilities.Reports.reportdata`? I get the same error from a number of other modules including:

- `reports.py`
- `ReportPageLogic.py`
- `perspective.py`
- `ReportDialog.py`
- `FittingPerspective.py`
- `CorfuncPerspective.py`

Reply:
Think this was an automatic refactoring error that happened when I moved things into reports

Reply:
latest push should fix this